### DISCUSSION TITLE
`padding-bottom` for `<Paragraph>`

### DISCUSSION THREAD

**Discussion by pr-apes** [speedata/publisher#471] [Q&A]
@pgundlach,

I have a `<Paragraph>` inside a `<Textblock>` (which is inside a `<Frame>`).

Using `<DefineTextformat>`, I can have `padding-top`.

`<Paragraph>` allows `padding-left` and `padding-left` and `padding-right`.

But how can I get `padding-bottom` for `<Paragraph>` (or for `<DefineTextformat>`)?

BTW, I think it might make sense to allow a global `padding` attribute (which would give values for all [four] paddings).

Many thanks for your help.

**Comment by pgundlach** [comment]
I'll have a look. I don't think it is possible at the moment.

**Reply by pr-apes** [reply]
Ok, I have added another paragraph with a narrow non-breaking space.
